### DISCUSSION TITLE
pytket install on ARM - aarch64?

### DISCUSSION THREAD

**Discussion by lifesuh** [CQCL/pytket#168] [Q&A] [answered]
$ lscpu
Architecture:        aarch64

$ pip --version
pip 22.2.2 ... (python 3.9)

$ pip install pytket
ERROR: Could not find a version that satisfies the requirement pytket (from versions: none)
ERROR: No matching distribution found for pytket
$ 

I can install qiskit on the ARM aarch64 architecture.
However, as shown above, doesn't tket/pytket support the ARM aarch64 architecture?

Thanks,

**Comment by cqc-alec** [comment, marked as answer]
Hi, pytket wheels are not currently distributed for the linux aarch64 architecture. You can see the supported OS/architecture combinations here: https://pypi.org/project/pytket/#files

**Reply by lifesuh** [reply]
Thanks for your answer.

Do you have a plan to distribute pytket wheels for the linux aarch64 architecture?

If not, how can I compile/build the pytket on the aarch64 architecture? source, doc, etc...

Thanks for your help,

**Reply by cqc-alec** [reply]
Hi, there are no current plans, but there is an open feature request: https://github.com/CQCL/tket/issues/359
Since this isn't a supported platform on github actions it isn't that easy to add it to the CI builds.
In principle it should be possible to build it yourself, adapting the insructions [here](https://github.com/CQCL/tket#how-to-build-tket-and-pytket). But as far as I know this hasn't yet been tried.

**Reply by lifesuh** [reply]
Thanks so much Alec,
Very helpful...
I will try it and let you know if any issues.

Thanks again
